codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/DafnyToAwsSdkShapeVisitor.java

```diff
@@ -132,20 +132,17 @@ public String blobShape(final BlobShape shape) {
       ? "input"
       : dataSource;
     return """
-    func () []byte {
-    b := []byte{}
-    if %s == nil {
-        return nil
-    }
-    for i := dafny.Iterate(%s) ; ; {
-        val, ok := i()
-        if !ok {
-            return b
-        } else {
-            b = append(b, val.(byte))
-        }
+      func () []byte {
+      if %s == nil {
+          return nil
+      }
+      a := input.(dafny.Sequence).ToArray().(dafny.GoNativeArray);
+    b := make([]byte, 0, a.Length())
```

Comment:
Just reminding myself that the default value when `b := []byte{}` and `b := make([]byte, ...)` is same. So, there should be no change in default value.

Reply:
`b := []byte{}` creates an empty slice, so the concept of default value doesn't apply.